src/styles/blocks/container.scss

```
@@ -0,0 +1,31 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  gap: 46px 48px;
+  grid-template-rows: repeat(2, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+}
+
+@media (min-width: 300px) {
+  .container {
+  grid-template-columns: repeat(1, 1fr);
+  }
```

Fix code style everywhere
```suggestion
  .container {
    grid-template-columns: repeat(1, 1fr);
  }
```